test/index.js

```diff
@@ -106,20 +84,34 @@ describe('Vero', function(){
         .track({ event: 'event' })
         .error('Bad Request', done);
     });
+
+    it('should unsubscribe when the event name is "unsubscribe"', function(done){
```

Comment:
Should it only match `^[uU]nsubscribe$`? right now it'll match e.g. `user canceled unsubscribe`.

Can you either fix the regex or clarify the conditions under which it will send to the unsubscribe endpoint?